### DISCUSSION TITLE
Error in Emailjs

### DISCUSSION THREAD

**Discussion by xXxAcexXx** [emailjs-com/help#18] [Q&A] [answered]
my emailjs was working in sending or receiving for almost a month now , but now suddenly it doesn't define the emailjs using the given html to define it which is <script type="text/javascript" 
    	src="https://cdn.jsdelivr.net/npm/@emailjs/browser@3/dist/email.min.js">
    </script>
	<script type="text/javascript">
		(function(){
			emailjs.init("User-id");
		})();
	</script>(index):25 Uncaught ReferenceError: emailjs is not defined
    at (index):25:7
    at (index):26:7 , i need support in this , since this is on the production website causing me alot of money.

**Comment by xr0master** [comment]
The code looks ok. There hasn't been a new SDK release lately. You suggest you check your site for other errors that may affect the running of scripts.

**Reply by xXxAcexXx** [reply]
Thank you for the quick reply :D , 
  i haven't changed the code for a week or so and it was working up until yesterday ,
what could be the cause of it , did the definition link change or so ?

**Reply by xr0master** [reply]
You can copy the link to your browser to see if the file opens without issues in your country. Works for me.
Or click below
https://cdn.jsdelivr.net/npm/@emailjs/browser@3/dist/email.min.js

**Reply by xr0master** [reply]
I can also suggest you not use the `init` method, but pass the public key as the fourth parameter in the `send[Form]` method.

**Reply by xXxAcexXx** [reply]
its true like you said , they have closed the support for my country , thats why .

**Reply by xXxAcexXx** [reply]
thank you for responding , but now i am 100% sure its blocked here ,maybe its the country or maybe my website in specific , how can i reach the support

**Reply by xr0master** [reply]
It's hard for me to say why the cdn.jsdelivr.net service is blocked in your country. You can download the script and add it to your site as your own script.
If you are using npm and builder, then just install the package.

Please note that the EmailJS service does not restrict you.

**Reply by xr0master** [reply, marked as answer]
Or you can use another CDN like unpkg.
https://unpkg.com/@emailjs/browser@3/dist/email.min.js
